### ISSUE TITLE
Feature request: Poll Now, please

### ISSUE THREAD

**Issue by heikoherold** [acristescu/OnlineGo#35]
Hi, low prio Feature Request for bad networking environment, also useful right after a change to/from a bad wifi: poll the game status immediately after manual trigger.
A nice trigger would be touch-clicking the upper right online-go icon, where the number of games with your turn is displayed. Give a minimal feedback; a rotating icon or something similar, at trigger happened until success or failure of the poll (or app is sent to background, of course).
Thanks

**Comment by benjaminpjones** [comment]
Maybe [Pull to Refresh](https://uxplanet.org/pull-to-refresh-ui-pattern-42a85f671cdf)? The home page reminds me a lot of an inbox.

**Comment by acristescu** [comment]
There isn't much need for this though, the app is always subscribed to the server changes through Websockets. It would complicate the main screen (which is too complex already).

**Comment by benjaminpjones** [comment]
I see. Is there any reason to keep this issue open then? Seems like it's a "Won't Fix"

**Comment by acristescu** [comment]
Not really, except for the fact that I don't have time to clean them up...
